package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "identity-reporting",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "test": "run-p test:*",
+    "test:typecheck": "tsc",
+    "test:lint": "eslint src --ext .js",
+    "test:unit": "mocha -r esbuild-register 'src/**/*.test.js'"
+  },
+  "dependencies": {
+    "d3-time-format": "^4.0.0",
+    "htm": "^3.1.0"
```

Comment:
Not in use currently. Fine to keep if you'd planned to use it in the future though.

Reply:
Good catch, removed in d11e28bf43daf06668fb4312f81b8e85f104c857